app/controllers/news_controller.rb

```diff
@@ -19,8 +18,10 @@ def create
   end
 
   private
-    # Only allow a list of trusted parameters through.
+    def get_user
+      @user = User.find(params[:user_id])
```

Comment:
if user_id will be incorrect there will be critical error. Better to use find_by and check user if nil to return correct error response